server/server.go

```diff
@@ -126,6 +130,45 @@ func (s *Server) StartEtcd(apiHandler http.Handler) error {
 		etcdCfg.UserHandlers[pdAPIPrefix] = apiHandler
 	}
 
+	var remoteClusterID uint64
```

Comment:
Good idea, I will do a clean up in next PR, move all etcd helper functions to 'pkg/etcdutil'.